.gitignore

```diff
@@ -22,3 +22,6 @@ third_party/lzma.js/lzma-native.exe
 tools/optimizer_build/
 
 .DS_Store
+
+.coverage*
+tools/coverage/
```

Comment:
Is it creating data here?   Is there a better place that is not in the source tree?

Reply:
Yes. As for a better place that's not in the source tree, it needs to be a consistent place since we need to read back the data later. I would use one of the emscripten cache directories, but we can't load `shared.py` to use that machinery. I think it's the easiest to just leave it in the source tree and ignore it. We already have `optimizer_build` doing the same thing anyways.